src/bo4e/cases.py

```diff
@@ -1,17 +1,25 @@
+"""
+Small module to convert snake_case to CamelCase and vice versa.
```

Comment:
```suggestion
Small module to convert snake_case to UpperCamelCase and vice versa.
```
oder meinst du "lowerCamelCase"?

Reply:
den Standard für Javascript Variablen. Also lower camelCase

Reply:
aa9718eae35a0f8f10c68fe6447a2d331f636f2b
done